.github/workflows/prettier.yaml

```diff
@@ -0,0 +1,25 @@
+name: Prettier
+# minimally modified from https://github.com/creyD/prettier_action#example-2-using-the-only_changed-or-same_commit-option-on-pr
+
+on:
+  pull_request:
+    branches: 
+      - main
+
+jobs:
+  prettier:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          # Make sure the actual branch is checked out when running on pull requests
+          ref: ${{ github.head_ref }}
+          # This is important to fetch the changes to the previous commit
+          fetch-depth: 0
+
+      - name: Prettify code
+        uses: creyD/prettier_action@v4.3
```

Comment:
May want to pair this with a Dependabot config like https://github.com/w3c/aria-practices/blob/83d1329687a9f3e35737ff1a49bb2678eab02c15/.github/dependabot.yml#L11 to keep this Action up-to-date

Reply:
Thanks, @nschonni. I've made a note. Let's see if this actually makes it first though :smile: